node/blockchain/inner_models/signed_change_request_message/genesis.py

```diff
@@ -42,8 +42,9 @@ def create_from_alpha_account_root_file(
     ):
         accounts = {}
         for account_number, account_state in account_root_file.items():
-            accounts[account_number] = AlphaAccount(
-                balance=account_state['balance'], balance_lock=account_state['balance_lock']
+            alpha_account = AlphaAccount(
+                balance=account_state['balance'], balance_lock=account_state['balance_lock'].lower()
```

Comment:
alpha account must contain data as is, because it something that is sign and also we must provide binary equality with the original account root file.

therefore you should not lower input here, but put it as is. Instead you allow upper case hex digits for `AlphaAccount`

Reply:
I added AlphaAccountLock, AlphaAccountNumber classes. Is it what you mean?